cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,85 @@
+= VPC Upgrade to Private Space 
+
+//Overview
+CloudHub VPC upgrade is an out-of-the-box tool and guided UI experience for an infrastructure upgrade to accelerate the migration of a VPC into a Private Space.
+
+// Benefits
+This tool enables you to provision the infrastructure of a CloudHb 2.0 Private Space by cloning the existing configurations of eligible CloubHub VPCs, and carrying over any existing network connections, including TGWs and VPNs. 
+
+At the same time, this UI experience reduces migration time and costs by eliminating a two- to-three-month toil to obtain all the necessary infrastructure and security approvals on new network connections and CIDR blocks. 
+
+
+== Understand Eligibility
+//Which VPCs are eligible for upgrade
+
+Due to their architectural differences and distinct interfaces, CloudHub 2.0 doesn't support some features in the same way CloudHub does. Follow this eligibility check to confirm which VPCs are eligible for upgrade:
+
+Availability of IPs::
++
+Your VPC must have at least 25% free space, and 100 or more available IP addresses per subnet to be eligible for upgrade.
+
+Deprecated Availability Zones (AZs)::
++
+VPCs using deprecated AZs are not eligible for upgrade.
+
+Legacy VPNs::
++
+VPCs with legacy VPNs configured are not eligible for upgrade.
+
+Direct Connect or VPC Peering:: 
+If your VPC has Direct Connect or VPC Peering configured in their VPC, migrate to Transit Gateway before upgrading.
+
+TLS 1.1 ciphers::
++
+The following ciphers can't be carried over from CloudHub to CloudHub 2.0:
++
+* ECDHE-ECDSA-AES128-SHA1
+* ECDHE-ECDSA-AES256-SHA1
+* ECDHE-RSA-AES128-SHA1
+* ECDHE-RSA-AES256-SHA1
+
+
+== Upgrade your VPC via Anypoint Platform
+
+//Steps to upgrade
+
+. From *Anypoint Platform*, select *Runtime Manager* > *Upgrade*.
++
+Alternatively, select *Runtime Manager* > *What's New* to read about how the VPC upgrade works, and click *Continue to Upgrade* to start upgrading your VPCs.
++ 
+The *Upgrade to CloudHub 2.0* window lists all VPCs that are eligible for upgrade.
```

Comment:
Confusing phrasing - the window also lists VPCs that are not eligible (not just the eligible ones).